src/states/DefenseState.js

```diff
@@ -65,17 +65,6 @@ class DefenseState extends React.Component {
                 clearTimeout(this.timer);
                 clearInterval(this.timerUpdate);
             }
-        } else {
```

Comment:
This was removing auto-starting the timer when changing settings due to the mismtch between `currentTime`/`currentBonus` & `settings.time/extraTime`.